lib/class-wp-theme-json-gutenberg.php

```diff
@@ -465,18 +465,20 @@ class WP_Theme_JSON_Gutenberg {
 	/**
 	 * Defines which pseudo selectors are enabled for which elements.
 	 *
-	 * The order of the selectors should be: visited, hover, focus, active.
-	 * This is to ensure that 'visited' has the lowest specificity
-	 * and the other selectors can always overwrite it.
+	 * The order of the selectors should be: link, any-link, visited, hover, focus, active.
+	 * This is to ensure the user action (hover, focus and active) styles have a higher
+	 * specificity than the visited styles, which in turn have a higher specificity than
+	 * the unvisited styles.
 	 *
 	 * See https://core.trac.wordpress.org/ticket/56928.
 	 * Note: this will affect both top-level and block-level elements.
 	 *
 	 * @since 6.1.0
+	 * @since 6.2.0 Added support for ':link' and ':any-link'.
```

Comment:
```suggestion
	 * @since 6.2.0 Added support for `:link` and `:any-link`.
```

In core we use this format. WDYT @oandregal ?

Reply:
Sounds good. I copied that comment from core directly and it [does not use the backtick](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-theme-json.php#L478). Though I see other uses of it, so I went ahead and applied your suggestion. It'll be fixed in the next backport cycle.